app/models/claim_review.rb

```diff
@@ -104,6 +111,19 @@ def find_request_issue_by_description(description)
     request_issues.find { |reqi| reqi.description == description }
   end
 
+  def search_table_ui_hash
+    {
+      claim_id: id,
+      veteran_file_number: veteran_file_number,
+      veteran_full_name: Veteran.find_by(file_number: veteran_file_number).name.formatted(:readable_full),
+      ep_codes: ep_codes,
+      claimant_names: claimants.map(&:name),
+      review_type: type_of_claim_review(self.class),
```

Comment:
The private method here is overkill. No need for if/else statements:

```suggestion
       review_type: type.underscore,
```

see: see: https://apidock.com/rails/String/underscore

Reply:
awesome, i was looking for something exactly like this!